packages/catalog-process/src/services/readModelService.ts

```diff
@@ -424,13 +424,22 @@ export function readModelServiceBuilder(
         .find((d) => d.id === descriptorId)
         ?.docs.find((d) => d.id === documentId);
     },
-    async listAgreements(
-      eservicesIds: EServiceId[],
-      consumersIds: TenantId[],
-      producersIds: TenantId[],
-      states: AgreementState[],
-      descriptorId?: DescriptorId | undefined
-    ): Promise<Agreement[]> {
+    // eslint-disable-next-line max-params
```

Comment:
Now that the param is only one, this is not needed anymore

Reply:
Thanks! I forgot to remove it.